Automattic/jetpack
# Enhance Media Extractor's extract method to support alt_text

As part of the Jetpack Search product we would like to support inclusion of alt_text in the indexed content. This patch updates the Media Extractor with an option so that the extracted alt_text from images is returned in the structured result.

https://github.com/Automattic/jetpack/issues/19273

See D71137 for full scope of related changes, including usage.
See D71433 for WP.com changes

#### Changes proposed in this Pull Request:
* Enhance extract method with support for including alt_text.

#### Does this pull request change what data or activity we track or use?
* No 

#### Testing instructions:

Unit Tests added to test new behavior of extract, they also verify existing behavior is not impacted.

* Jetpack_Media_Meta_Extractor::extract( BLOG_ID, POST_ID, Jetpack_Media_Meta_Extractor::ALL, true );
* Verify that alt_text is in the image results for featured image and any <img> tags with alt="" attribute
